uefi-test-runner/src/proto/ata/pass_thru.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use uefi::boot;
+use uefi::boot::{OpenProtocolAttributes, OpenProtocolParams};
+use uefi::proto::ata::pass_thru::AtaPassThru;
+use uefi::proto::ata::AtaRequestBuilder;
+
+pub fn test() {
+    info!("Running ATA PassThru tests");
+
+    assert!(try_find_drive());
+}
+
+fn try_find_drive() -> bool {
```

Comment:
nit: `try_` typically comes with a `Result` return type. I think `drive_is_present` might be a better name